debian/control

```diff
@@ -2,33 +2,30 @@ Source: libtpms
 Maintainer: Stefan Berger <stefanb@linux.vnet.ibm.com>
 Section: libs
 Priority: optional
-Standards-Version: 3.9.7
-Build-Depends:
- autoconf,
- automake,
- autotools-dev,
- debhelper (>= 9),
- dh-autoreconf,
- dh-exec,
- gawk,
- libssl-dev,
- libtool,
- pkg-config
-
-Package: libtpms0
-Architecture: any
-Multi-Arch: same
-Depends: openssl,
-	 ${shlibs:Depends},
-	 ${misc:Pre-Depends},
-	 ${misc:Depends}
-Description: TPM emulation library
- Libtpms is a library that provides TPM functionality.
+Standards-Version: 4.5.1
+Rules-Requires-Root: no
+Build-Depends: debhelper (>= 10),
+               dh-exec,
+               gawk,
+               libssl-dev,
+               libtool,
+               pkg-config
 
 Package: libtpms-dev
 Architecture: any
 Section: libdevel
-Depends: ${misc:Depends}, libtpms0 (= ${binary:Version})
+Depends: libtpms0 (= ${binary:Version}),
+         ${misc:Depends}
```

Comment:
Package and dependency lists have been cleaned up and sorted as per `wrap-and-sort(1)`.